test/unit-test-helpers/mock_responses/vast_wrapper_2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?><VAST version="2.0">
+  <Ad id="wrapper-parent-2">
+  <Wrapper>
+    <AdTitle>wrapperParentAd2</AdTitle>
+    <VASTAdTagURI>vastad.xml</VASTAdTagURI>
+    <Impression>impressionUrl</Impression>
+    <Error>errorUrl</Error>
+	<Creatives>
+		<Creative AdID="wrapper-parent-creative-id-2">
+			<Linear>
+				<TrackingEvents>
+          <Tracking event="start">startUrl</Tracking>
```

Comment:
would it be useful to call this startUrl2 or something, so you know you aren't firing the same events as in the first wrapper file?